newroutes/RequestSchema.js

```diff
@@ -5,17 +5,20 @@ const FrontMatterSchema = Joi.object({
   permalink: Joi.string().required(),
 })
 
```

Comment:
Could we name this file with camelCase to fit in with the convention, i.e. `requestSchema.js` - and could we create a new `validators` folder